src/netcdf_writer.jl

```diff
@@ -141,7 +141,10 @@ function NetCDFWriter(
 
             num_vpts = Meshes.nelements(Grids.vertical_topology(space).mesh)
 
-            @warn "Disabling vertical interpolation, the provided number of points is ignored (using $num_vpts)"
+            # For any configuration, it is reasonable to assume that the last
+            # value of `num_pts` is the number of vertical points
+            (last(num_points) != num_vpts) &&
```

Comment:
```suggestion
            last(num_points) != num_vpts &&
```
I don't think the parenthesis are needed